kas/opt/meta-hailo.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2024
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+header:
+  version: 15
+
+local_conf_header:
+  hailo: |
+    IOT2050_META_HAILO = "1"
+  hailo_override: |
+    OVERRIDES .= ":meta-hailo"
+
+repos:
+  meta-hailo:
+    path: ${TOPDIR}/../
+    layers:
+      meta-hailo:
```

Comment:
Just append meta-hailo as layer to meta-iot2050.

Reply:
Done